kafka2/getmessages_kafka2.py

```diff
@@ -887,7 +914,7 @@ def parse_json_message(messages):
 
 
 def parse_json_message_single(message):
-    message = json.loads(json.dumps(message))
+    # message = json.loads(json.dumps(message))
```

Comment:
why need to comment out message = json.loads(json.dumps(message))

Reply:
this is no need and take more time.